libgnucash/quotes/gnc-fq-dump.in

```diff
@@ -207,6 +218,7 @@ gnc-fq-dump	- Print out data from the F::Q module
     gnc-fq-dump europe 48406.PA 13000.PA
     gnc-fq-dump vwd 632034
     gnc-fq-dump ftportfolios FKYGTX
+    gnc-fq-dump currency USD AUD
```

Comment:
I would put it at the beginning of the list as I estimate it globally seen as the most often used source.

Reply:
Thanks @fellen. I have moved the currency example to the front and added a couple more.